zerver/lib/email_mirror.py

```diff
@@ -23,13 +23,19 @@
 from zerver.lib.rate_limiter import RateLimitedObject
 from zerver.lib.exceptions import RateLimited
 from zerver.models import Stream, Recipient, MissedMessageEmailAddress, \
-    get_display_recipient, \
+    get_display_recipient, MAX_MESSAGE_LENGTH, MAX_TOPIC_NAME_LENGTH, \
     Message, Realm, UserProfile, get_system_bot, get_user, get_stream_by_id_in_realm
 
 from zproject.backends import is_user_active
 
 logger = logging.getLogger(__name__)
 
+def truncate_body(body: str) -> str:
```

Comment:
Let's actually put these in `message.py`.

Reply:
Er, let's put these in a new file called `zerver/lib/send_message.py`, actually.

Reply:
Okay

Reply:
Should truncate_content() be also moved in message.py